src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenRegistration.cs

```diff
@@ -53,11 +53,12 @@ public ValueTask DisposeAsync()
                 default;
         }
 
-        /// <summary>
-        /// Gets the <see cref="CancellationToken"/> with which this registration is associated.  If the
-        /// registration isn't associated with a token (such as after the registration has been disposed),
+        /// <summary>Gets the <see cref="CancellationToken"/> with which this registration is associated.</summary>
```

Comment:
Should these updates be included in `dotnet-api-docs` repo as well? cc: @carlossanlop

Reply:
Yes:
https://github.com/dotnet/dotnet-api-docs/pull/4186
:Smile:

Reply:
Awesome 🚀